.cbmc-batch/jobs/aws_priority_queue_s_remove_node/Makefile

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+###########
+include ../Makefile.aws_array_list
+
+###########
+#
+# Runtime: 500s
+
+## This is here for the backpointer array list which contains pointers
+## to aws_priority_queue_node elements. As pointers are 64bits, it is
+## adequate to unroll the memcpy loop twice.
+UNWINDSET += memcpy_using_uint64_impl.0:2
```

Comment:
@karkhaz @tautschnig is it reasonable to assume that pointers are 64bit sized? This unwind here is adequate because it holds when pointer sizes are <= 64 bits.